src/components/sbb-divider/sbb-divider.scss

```diff
@@ -0,0 +1,25 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include host-component-properties;
+
+:host {
+  --divider-color: var(--sbb-color-cloud-default);
+  --border-width: var(--sbb-border-width-1x);
+}
+
+:host(.sbb-divider) {
+  display: block;
+}
+
+:host(.sbb-divider--horizontal) {
+  border-top: var(--border-width) solid var(--divider-color);
+}
+
+:host(.sbb-divider--vertical) {
+  height: 100%;
+  border-left: var(--border-width) solid var(--divider-color);
+}
+
+:host(.sbb-divider--negative) {
```

Comment:
Move this directly below the `:host` rule and just change the css variable:
  --divider-color: var(--sbb-color-iron-default);

Reply:
Right, good catch 👍